website_cms_theme/views/assets.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!-- Assets, less, js -->
+    <template id="website_cms_them" name="Website CMS Theme"
+        inherit_id="website.assets_frontend">
+        <xpath expr="link[last()]" position="after">
+            <link href="/website_cms_theme/static/src/less/layouts/form.less"
+                rel="stylesheet" type="text/less"/>
+            <link href="/website_cms_theme/static/src/less/assets/varriables.less"
```

Comment:
this should be `variables.less` and the filename should change accordingly, of course ;)